migrations/20211123172816_set-additional-conditions-version.js

```diff
@@ -0,0 +1,13 @@
+exports.up = async function up(knex) {
+  await knex.schema.raw(`
+    update licences set additional_conditions_version = 1
+    where licence  -> 'licenceConditions' -> 'additional' is not null
+`)
+}
+
+exports.down = async function down(knex) {
+  await knex.schema.raw(`
+  update licences set additional_conditions_version = null
+  where licence  -> 'licenceConditions' -> 'additional' is not null
```

Comment:
Trivial observation: Don't think you need a restriction all additional_conditions_version were null before 'up'

Reply:
I've removed the `where`